pkg/deploy/controller/deployment/controller.go

```diff
@@ -107,6 +107,11 @@ func (c *DeploymentController) Handle(deployment *kapi.ReplicationController) er
 			// Don't try and re-create the deployer pod.
 			break
 		}
+
+		if _, ok := deployment.Annotations[deployapi.DeploymentIgnorePodAnnotation]; ok {
```

Comment:
This is mostly for testability - it might have use if we integrated a
third party deployment engine (or a third party controller).
Otherwise not relevant to most users.